### PR TITLE
Added food101, shakespeare and Penn TreeBank datasets

### PR DESCRIPTION
**Pull request by gaurav272333**

### CODE REVIEW

**Comment by vbvg2008** [comment]
add tests :)

File: fastestimator/dataset/data/penn_treebank.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+import wget
+
+from fastestimator.dataset.numpy_dataset import NumpyDataset
+from fastestimator.util.wget_util import bar_custom, callback_progress
+
+wget.callback_progress = callback_progress
+
+
+def load_data(root_dir: Optional[str] = None,
+              seq_length: int = 64) -> Tuple[NumpyDataset, NumpyDataset, NumpyDataset, List]:
```
Comment (TortoiseHam) [inline fastestimator/dataset/data/penn_treebank.py:29]: add typing to the List. is it a List[str] ?

File: fastestimator/dataset/data/shakespeare.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+import numpy as np
+import wget
+
+from fastestimator.dataset.numpy_dataset import NumpyDataset
+from fastestimator.util.wget_util import bar_custom, callback_progress
+
+wget.callback_progress = callback_progress
+
+
+def load_data(root_dir: Optional[str] = None, seq_length: int = 100) -> Tuple[NumpyDataset, List]:
```
Comment (TortoiseHam) [inline fastestimator/dataset/data/shakespeare.py:28]: add typing to the return List

File: fastestimator/dataset/data/food101.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+import pandas as pd
+import wget
+
+from fastestimator.dataset.csv_dataset import CSVDataset
+from fastestimator.util.wget_util import bar_custom, callback_progress
+
+wget.callback_progress = callback_progress
+
+
+def _create_csv(images: List, label_dict: Dict, csv_path: str) -> None:
```
Comment (TortoiseHam) [inline fastestimator/dataset/data/food101.py:30]: typing on List and Dict

**Review by vbvg2008 — APPROVED** [review]

**Pull request merged**